moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -1143,33 +1205,33 @@ void PlanningSceneMonitor::onStateUpdate(const sensor_msgs::JointStateConstPtr&
     updateSceneWithCurrentState();
 }
 
-void PlanningSceneMonitor::stateUpdateTimerCallback(const ros::WallTimerEvent& /*unused*/)
+void PlanningSceneMonitor::stateUpdateTimerCallback(/*const ros::WallTimerEvent& event*/)
 {
   if (state_update_pending_)
   {
     bool update = false;
 
-    const ros::WallTime& n = ros::WallTime::now();
-    ros::WallDuration dt = n - last_robot_state_update_wall_time_;
+    std::chrono::system_clock::time_point n = std::chrono::system_clock::now();
+    std::chrono::duration<double> dt = n - last_robot_state_update_wall_time_;
 
     {
       // lock for access to dt_state_update_ and state_update_pending_
       boost::mutex::scoped_lock lock(state_pending_mutex_);
-      if (state_update_pending_ && dt >= dt_state_update_)
+      if (state_update_pending_ && dt.count() >= dt_state_update_.seconds())
       {
         state_update_pending_ = false;
-        last_robot_state_update_wall_time_ = ros::WallTime::now();
+        last_robot_state_update_wall_time_ = std::chrono::system_clock::now();
+        auto sec = std::chrono::duration<double>(last_robot_state_update_wall_time_.time_since_epoch()).count();
         update = true;
-        ROS_DEBUG_STREAM_NAMED(LOGNAME,
-                               "performPendingStateUpdate: " << fmod(last_robot_state_update_wall_time_.toSec(), 10));
+        RCLCPP_DEBUG(node_->get_logger(), "performPendingStateUpdate: %f", fmod(sec, 10));
       }
     }
 
     // run the state update with state_pending_mutex_ unlocked
     if (update)
     {
       updateSceneWithCurrentState();
-      ROS_DEBUG_NAMED(LOGNAME, "performPendingStateUpdate done");
+      RCLCPP_DEBUG(node_->get_logger(), "performPendingStateUpdate done");
```

Comment:
```suggestion
      RCLCPP_DEBUG(LOGGER, "performPendingStateUpdate done");
```
Same for all logger calls